doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -52,6 +58,8 @@
               help="Quay ORGANIZATION to mirror into.\ndefault=openshift-release-dev")
 @click.option("--repository", metavar="REPO", required=False, default="ocp-v4.0-art-dev",
               help="Quay REPOSITORY in ORGANIZATION to mirror into.\ndefault=ocp-v4.0-art-dev")
+@click.option("--private-repository", metavar="REPO", required=False, default="ocp-v4.0-art-dev-priv",
+              help="Private Quay REPOSITORY in ORGANIZATION to mirror into.\ndefault=ocp-v4.0-art-dev-priv")
```

Comment:
I think the repo name ocp-v4.0-art-dev-priv won't change so no need to be a parameter, can be a constant value or a hardcode

Reply:
I followed the same pattern for the public quay. So I think we can raise a follow up PR if we want to convert them to constants.